docs/reasons.rst

```diff
@@ -0,0 +1,120 @@
+.. _reasons:
+
+===============
+Why PyScaffold?
+===============
+
+|
+
+Stable and battle-proven
+    PyScaffold was created in 2014 to make the developers' life easier.
+    Since then it have been used to create awesome Python packages for data
+    science, industrial automation, academic research, telecom, web development
+    and many other sectors.
+
+Constantly evolving
+    Stability should not come with the price of stagnation.
+    Throughout its existence, PyScaffold have adapted itself to better solve
+    the needs of its users and evolved to accommodate the best practices and
+    standards of the Python ecosystem.
+    In every single major release we offered a clear update path for our users
+    and tried to automate things as much as possible, so everyone can benefit
+    from PyScaffold's improvements without being afraid of breaking things.
+
+Thoroughly tested
+    PyScaffold have an extensive automated test suite that runs for all major
+    operating systems and versions of Python in every commit or pull request.
+    Moreover, PyScaffold is used by its maintainers in their day-to-day
+    programming and for all PyScaffold's extensions, being constantly
+    scrutinized by human beings.
```

Comment:
I don't completely understand the meaning of "... and for all PyScaffold's extensions, being constantly scrutinized by human beings". So PyScaffold is used even for the extensions which are then again tested by human beings?

Reply:
I wanted to say that PyScaffold is not only tested in an automated way, but also have people with their eyes on it... Let me see how to rephrase it.